modules/local/busco-auto-lineage.nf

```diff
@@ -0,0 +1,107 @@
+process BUSCO {
```

Comment:
there is already an nf-core BUSCO module - how is this different or needed from that one?

Reply:
Sorry, the auto-lineage one is no longer used. But as mentioned above, I have edited the busco one so I should probably just move it to local so as not to be confusing.